website/app/templates/head.hbs

```diff
@@ -0,0 +1,90 @@
+{{!
+
+Per https://github.com/shipshapecode/ember-meta#defining-your-own-headhbs this is a place where we can customize what information is injected by the ember-meta addon
+
+We've made some small customizations to simplify this compared to the default template
+
+}}
+
+{{! Begin General }}
```

Comment:
I find it hard to see when things begin and end, with this simple comments. can you maybe make them move evident (eg. ALL UPPERCASE) and separate them more from the code?